source/declarations.tex

```diff
@@ -2084,6 +2090,8 @@
 \indextext{specifier|)}%
 
 \rSec1[dcl.decl]{Declarators}%
+
+\rSec2[dcl.decl.general]{General}%
 \indextext{declarator|(}
```

Comment:
This is not nice; the "declarator" start of index range should probably be before the "General" section. But maybe this is not the right time to fiddle with this.

Reply:
We should fix this for the `master` branch. I think this is fine for C++20.